_locales/en/messages.json

```diff
@@ -646,6 +646,12 @@
     "configurationBatteryScale": {
         "message": "Voltage Scale"
     },
+    "configurationVbatPidCompensation": {
+        "message": "Vbat PID Compensation"
+    },
+    "configurationVbatPidCompensationHelp": {
+        "message": "Increase the PID values to compensate when Vbat gets lower. This will give more constant flight characteristics throughout the flight."
```

Comment:
I can't find a reference to `calculateBatteryPercentage`. I must really be seeing things. :-(

```
~/git/betaflight$ grep calculateBatteryPercentage src/main/flight/mixer.c 
~/git/betaflight$ 
```